istioctl/cmd/proxyconfig.go

```diff
@@ -43,6 +43,7 @@ const (
 	prometheusMergedOutput = "prom-merged"
 
 	defaultProxyAdminPort = 15000
+	loggerIDs             = "https://github.com/envoyproxy/envoy/blob/a62fa454ffa4c28bec01d37788a38e71a0e230d3//source/common/common/logger.h#L29"
```

Comment:
Is this the correct line? Should it be L35?

Reply:
Btw, where is this being used?

Reply:
Forgot deleting this, thanks